### PR TITLE
Prepare for Initial release

### PR DESCRIPTION
**Pull request by thegecko**
#### What it does

Prepare for initial release. Propose v1.0.0 and have it set as a preview release.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by colin-grant-work** [comment]
Is it planned to publish to both Open VSX and the VSCode Marketplace, or just Open VSX?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ jobs:
           node-version: 16.x
       - name: Publish
         run: |
-          npx ovsx@0.5.1 publish -i artifacts/*/*.vsix -p ${{secrets.OPEN_VSX_TOKEN}}
+          npx ovsx@0.8.0 publish -i artifacts/*/*.vsix -p ${{secrets.OPEN_VSX_TOKEN}}
```
Comment (colin-grant-work) [inline .github/workflows/ci.yml:65]: Is it definitely worth specifying the version, or could we go with a versionless invocation and trust latest?

**Review by colin-grant-work — APPROVED** [review]

**Comment by thegecko** [comment]
> Is it planned to publish to both Open VSX and the VSCode Marketplace, or just Open VSX?

Both

File: .github/workflows/ci.yml (hunk 1 above)
Comment (thegecko) [reply .github/workflows/ci.yml:65]: 0.8.0 has worked well for us, but I'm happy to go version-less.

**Pull request merged**